hypothesis-python/scripts/basic-test.sh

```diff
@@ -70,9 +71,9 @@ if [ "$(python -c 'import platform; print(platform.python_implementation())')" !
   HYPOTHESIS_DJANGO_USETZ=FALSE python -m tests.django.manage test tests.django
   pip uninstall -y django pytz
 
-  pip install numpy
+  pip install "$(grep 'numpy==' ../requirements/coverage.txt)"
```

Comment:
Instead of pinning to an old version of numpy for all versions, let's add an if-statement here so we use the last-compatible version for 3.7, and from coverage.txt for newer Pythons.